roles/crawler/tasks/configure-databases.yml

```diff
@@ -99,6 +110,12 @@
       lc_ctype: en_US.UTF-8
       template: "{{ fpsd_database_psql_env.PGDATABASE }}"
 
-  when: raw_schema_result|changed or raw_schema_tables_result|changed
+  - name: Setup TABLEFUNC extension.
+    postgresql_ext: name=tablefunc db="test{{ fpsd_database_psql_env.PGDATABASE }}"
+    register: postgres_test_extension
+    always_run: true
+    changed_when: false
+
+  when: raw_schema_result|changed or raw_schema_tables_result|changed or features_schema_result|changed
```

Comment:
So I've made this: `when: raw_schema_tables_result|changed or postgres_extension|skipped or  schemas|changed`